docs/src/content/docs/tr/reference/overrides.md

```diff
@@ -0,0 +1,375 @@
+
+## Bileşen Özellikleri
+
+Tüm bileşenler, mevcut sayfa hakkında bilgi içeren standart bir `Astro.props` nesnesine erişebilir.
+
+Özel bileşenlerinizi yazmak için Starlight'tan `Props` türünü içe aktarın:
+
+```astro
+---
+import type { Props } from '@astrojs/starlight/props';
+
+const { hasSidebar } = Astro.props;
+//      ^ Tür: boolean
+---
+```
+
+Bu, `Astro.props`'a erişirken otomatik tamamlamayı ve türleri sağlar.
+
+### Props
+
+Starlight, özel bileşenlerinize aşağıdaki özellikleri iletecektir.
+
+#### `dir`
+
+**Tür:** `'ltr' | 'rtl'`
+
+Sayfanın yazma yönü.
+
+#### `lang`
+
+**Tür:** `string`
+
+Bu sayfanın bölgesi için BCP-47 dil etiketi, örneğin `en`, `zh-CN`, veya `pt-BR`.
+
+#### `locale`
+
+**Tür:** `string | undefined`
+
+Bir dilin sunulduğu temel yol. Kök bölge dize sürümleri için `undefined` .
+
+#### `slug`
+
+**Tür:** `string`
+
+İçerik dosyasının adından oluşturulan bu sayfanın slug'ı.
+
+#### `id`
+
+**Tür:** `string`
+
+İçerik dosyası adına dayalı olarak bu sayfanın benzersiz kimliği.
+
+#### `isFallback`
+
+**Tür:** `true | undefined`
+
+Eğer bu sayfa, mevcut dilde çevrilmemişse ve varsayılan bölgeye ait içeriği kullanıyorsa `true`. Sadece çok dilli sitelerde kullanılır.
+
+#### `entryMeta`
+
+**Tür:** `{ dir: 'ltr' | 'rtl'; lang: string }`
+
+Sayfa içeriği için bölge metaverisi. Bir sayfa, fallback içerik kullanıyorsa, sayfa içeriğinin baş düzey bölge değerlerinden farklı olabilir.
+
+#### `entry`
+
+Mevcut sayfa için Astro içerik koleksiyonu girişi. 
+`entry.data` altındaki mevcut sayfa için frontmatter değerlerini içerir.
+
+```ts
+entry: {
+  data: {
+    title: string;
+    description: string | undefined;
+    // vb.
+  }
+}
+```
+
+Bu nesnenin şekli hakkında daha fazla bilgiyi [Astro'nun Koleksiyon Giriş Türü](https://docs.astro.build/en/reference/api-reference/#collection-entry-type) başvurusunda bulabilirsiniz.
+
+#### `sidebar`
+
+**Tür:** `SidebarEntry[]`
+
+Bu sayfa için site gezinme kenar çubuğu girişleri.
+
+#### `hasSidebar`
+
+**Tür:** `boolean`
+
+Bu sayfada kenar çubuğunun gösterilip gösterilmeyeceğini belirler.
+
+#### `pagination`
+
+**Tür:** `{ prev?: Link; next?: Link }`
+
+Kenar çubuğundaki önceki ve sonraki sayfa bağlantıları.
+
+#### `toc`
+
+**Tür:** `{ minHeadingLevel: number; maxHeadingLevel: number; items: TocItem[] } | undefined`
+
+Bu sayfa için içerik tablosu, açıkken.
+
+#### `headings`
+
+**Tür:** `{ depth: number; slug: string; text: string }[]`
+
+Mevcut sayfadaki tüm Markdown başlıklarının dizisi. 
+Starlight'ın yapılandırma seçeneklerini dikkate alan bir içerik tablosu bileşeni oluşturmak istiyorsanız, [`toc`](#toc) yerine kullanın.
+
+#### `lastUpdated`
+
+**Tür:** `Date | undefined`
+
+Etkinleştirilmişse, bu sayfanın son ne zaman güncellendiğini temsil eden JavaScript `Date` nesnesi.
+
+#### `editUrl`
+
+**Tür:** `URL | undefined`
+
+Eğer etkinse, bu sayfanın düzenlenebileceği adres için bir `URL` nesnesi.
+
+---
+
+## Components
```

Comment:
Should this be translated with "Bileşenler" like in the components guide?

Reply:
yup, my mistake